docs/MO_DG/prepare_model/convert_model/mxnet_specific/Convert_Style_Transfer_From_MXNet.md

```diff
@@ -2,6 +2,13 @@
 
 @sphinxdirective
 
+
+.. warning::
+
+   Note that OpenVINO support for Apache MXNet, is currently being deprecated and will be removed entirely in the future.
```

Comment:
```suggestion
   Note that OpenVINO support for Apache MXNet, is currently being deprecated and will be removed entirely in future releases.
```

Reply:
future releases might suggest "23.1 onwards" ... will stay with the more general wording not to introduce ambiguity.